makefile.ver

```diff
@@ -33,8 +33,8 @@ VERSION_DATA_FILE = $(WORKDIR)/daal_version_gen.h
 
 # content of version data file to be included in product
 define DAAL_VERSION_DATA
-// DO NOT EDIT: file is auto-generated on build time
-// DO NOT PUT THIS FILE TO SVN: file is auto-generated on build time
+// DO NOT PUT THIS FILE TO VERSIONING: file is auto-generated on build time
+// Product version is specified in `makefile.ver` file
 // Product version is specified in src/makefile.ver file
```

Comment:
Don't these two lines say the same?

Reply:
Thanks for noticing, was a problem with rebasing.